features/cmake/find_package/exported_targets_multiconfig/bye/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, CMake
+
+
+class ByeConan(ConanFile):
+    name = "bye"
+    version = "1.0"
+    settings = "os", "compiler", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "cmake_paths"
+    exports_sources = "src/*"
+    requires = "hello/1.0@user/channel"
+
+    def configure(self):
+        # it is also necessary to remove the VS runtime
```

Comment:
Why? Not clear why, this is the first time I see this. A runtime will indeed be used why remove it?

Reply:
I can't remember. Probably unnecessary, will review.

Reply:
Looks like it is unnecessary. Probably dead code from the beginnings of the investigation.